ui/src/main/js/distribution/channels/azure/AzureDistributionConfiguration.js

```diff
@@ -0,0 +1,79 @@
+import PropTypes from 'prop-types';
+import React, { useEffect } from 'react';
+import * as FieldModelUtilities from 'util/fieldModelUtilities';
+import TextInput from 'field/input/TextInput';
+import CheckboxInput from 'field/input/CheckboxInput';
+import { AZURE_DISTRIBUTION_FIELD_KEYS } from 'distribution/channels/azure/AzureModel';
+
+const AzureDistributionConfiguration = ({
+    data, setData, errors, readonly
+}) => {
+    useEffect(() => {
+        setData(FieldModelUtilities.updateFieldModelSingleValue(data, AZURE_DISTRIBUTION_FIELD_KEYS.workItemType, 'Task'));
+    }, []);
+    return (
+        <>
+            <CheckboxInput
+                name={AZURE_DISTRIBUTION_FIELD_KEYS.comment}
+                label="Comment on Work Items"
+                description="If selected, Alert will comment on Work Items it created when updates occur."
+                readOnly={readonly}
+                onChange={FieldModelUtilities.handleChange(data, setData)}
+                isChecked={FieldModelUtilities.getFieldModelBooleanValue(data, AZURE_DISTRIBUTION_FIELD_KEYS.comment)}
+                errorName={FieldModelUtilities.createFieldModelErrorKey(AZURE_DISTRIBUTION_FIELD_KEYS.comment)}
+                errorValue={errors[AZURE_DISTRIBUTION_FIELD_KEYS.comment]}
+            />
+            <TextInput
+                id={AZURE_DISTRIBUTION_FIELD_KEYS.project}
+                label="Azure Project"
+                description="The project name or id in Azure Boards."
+                name={AZURE_DISTRIBUTION_FIELD_KEYS.project}
+                required
+                onChange={FieldModelUtilities.handleChange(data, setData)}
+                value={FieldModelUtilities.getFieldModelSingleValueOrDefault(data, AZURE_DISTRIBUTION_FIELD_KEYS.project)}
```

Comment:
I'm seeing a lot of references to *OrDefault which is supposed to take in 3 arguments but only 2 are passed in. Was this by design? Additionally as a note (A bug I ran into), I tried using this function previously but ran into issues with the default being shown but not actually populating the fieldmodel object as it's just a getter.

Reply:
This was a copy paste error I will search for it.